lib/metasploit/framework/password_crackers/jtr/formatter.rb

```diff
@@ -67,6 +67,9 @@ def hash_to_jtr(cred)
       # with a sample hash of b31d032cfdcf47a399990a71e43c5d2a:144816. So this just outputs
       # The hash as *hash*: so that it is both JTR and hashcat compatible
       return "#{cred.private.data}:"
+    when /vnc/
+      # add a beggining * if one is missing
```

Comment:
```suggestion
      # add a beginning * if one is missing
```

Reply:
Fixed in https://github.com/rapid7/metasploit-framework/pull/16341/commits/1408ffbab7d8fa24fa0a6fd4ff6abcb7ca5dce04